src/main/java/Duke/Event.java

```diff
@@ -0,0 +1,20 @@
+package Duke;
+
+public class Event extends Task {
+    protected String at;
+
+    public Event(String name, String at) {
+        super(name);
+        this.at = at;
+    }
+
+    @Override
+    public String getParsedData() {
+        return "E" + "/" + String.valueOf(super.done) + "/" + super.name + "/" + this.at;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + at + ")";

```

Comment:
A little inconsistent with the this.at and at. Not that there is anything wrong.